File: Offer/C++/offer53.cc
#include<iostream>
#include<vector>
using namespace std;

class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int l = 0;
        int r = nums.size()-1;

        while(l <= r){
            int mid = (l+r) / 2;
            if(nums[mid] == mid){
                l = m +1;
            }
            else{
                r = m-1;
            }
        }
        return l;
    }
};

File: Offer/golang/Array/Offer53.go
package Array

func missingNumber(nums []int) int {
	res := make([]int, 0, len(nums))
	index := nums[len(nums)-1]
	for i := 0; i < index; i++ {
		res = append(res, nums[i])
	}
	for i := 0; i <
}
